CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ The versions follow [semantic versioning](https://semver.org).
   - Fennel (`.fnl`) (#638)
   - CommonJS (`.cjs`) (#632)
   - Qt .pro (`.pro`) (#632)
+  - [Textile](https://textile-lang.com/doc/textile-comments) (`.textile`) (#712)
```

Comment:
Could you please remove the link? It suffices if more information is given in the PR.